debian/packetfence.preinst

```diff
@@ -56,8 +56,12 @@ case "$1" in
             else
                 useradd -r -d "/usr/local/pf" -s /bin/sh -c "PacketFence" -M pf -g pf
             fi
+            usermod -u 6969 pf
```

Comment:
We shouldn't be using the static UID/GID based on our latest conversations unless there is another reason why  this should still apply